<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng nhập</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: rgb(70, 70, 70);
    padding: 6px 12px;
    font-weight: bold;
}</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hệ Thống Quản Lý Cho Thuê Truyện Và Sách</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #4CAF50;   /* Màu nền nút */
    color: white;                /* Màu chữ */
    border-radius: 10px;         /* Bo góc */
    padding: 6px 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color:E74C3C ;   /* Khi rê chuột */
}

QPushButton:pressed {
    background-color: #2e7031;   /* Khi nhấn giữ */
}</string>
   </property>
   <property name="text">
    <string>Đăng nhập</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #E74C3C;   /* Màu nền nút */
    color: white;                /* Màu chữ */
    border-radius: 10px;         /* Bo góc */
    padding: 6px 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color:E74C3C ;   /* Khi rê chuột */
}

QPushButton:pressed {
    background-color: #2e7031;   /* Khi nhấn giữ */
}</string>
   </property>
   <property name="text">
    <string>Thoát</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mật khẩu:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Tên đăng nhập:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
